gcc/rust/util/rust-unicode.cc

```diff
@@ -282,6 +284,13 @@ nfc_normalize (string_t s)
   return r;
 }
 
+Utf8String
+Utf8String::nfc_normalize () const
+{
+  string_t normalized = Rust::nfc_normalize (chars);
```

Comment:
I'm not too sure about the intent with the `normalized` variable here. Is it used somewhere ?

Reply:
Oh thanks, it is unused variable.